### DISCUSSION TITLE
Filtron fails to start during Searx installation

### DISCUSSION THREAD

**Discussion by nikodean1** [searx/searx#2893] [Installation issue: Official wiki]
![Searx Installation Failure](https://user-images.githubusercontent.com/19659265/125085839-a13d6580-e090-11eb-8259-afdae5d55f87.png)

Hi, i'm trying to figure out why Searx won't install. I followed the instructions exactly as stated by the official guide here: https://searx.github.io/searx/admin/installation.html

The OS is Ubuntu 20.04 and it is up-to-date

**Comment by allendema** [comment]
To me it seems like there is a lack of permission install Go binary and therefore Filtron cannot be run.
